pkg/broker/broker.go

```diff
@@ -240,8 +240,8 @@ func (a AnsibleBroker) Bootstrap() (*BootstrapResponse, error) {
 func addNameAndIDForSpec(specs []*apb.Spec, registryName string) {
 	for _, spec := range specs {
 		//need to make / a hyphen to allow for global uniqueness but still match spec.
-		spec.FQName = strings.Replace(fmt.Sprintf("%v-%v", registryName, spec.Image),
-			"/", "-", -1)
+		spec.FQName = strings.Replace(fmt.Sprintf("%v-%v", registryName,
+			strings.Replace(spec.Image, ":", "-", -1)), "/", "-", -1)
```

Comment:
Normally I'm ok with the embedded calls but it took me quite a bit to read this line with the 2 string replaces and the sprintf. Let's pull out the inner replacement:

```golang
imageName := strings.Replace(spec.Image, ":", "-", -1)
spec.FQName = strings.Replace(fmt.Sprintf("%v-%v", registryName, imageName), "/", "-", -1)
```